src/nerve_tools/configure_nerve.py

```diff
@@ -24,7 +24,9 @@
 from environment_tools.type_utils import compare_types
 from environment_tools.type_utils import convert_location_type
 from environment_tools.type_utils import get_current_location
-from paasta_tools.marathon_tools import get_services_running_here_for_nerve
```

Comment:
Are you going to remove `get_services_running_here_for_nerve` from paasta_tools too?